detox/src/artifacts/log/android/ADBLogcatRecording.js

```diff
@@ -26,26 +25,18 @@ class ADBLogcatRecording extends Artifact {
     this._waitUntilLogFileIsCreated = null;
   }
 
-  get hasRecordedFile() {
-    return !!this._waitUntilLogFileIsCreated;
-  }
-
   async doStart() {
     const pid = this.pid.get();
 
-    if (pid > 0) {
```

Comment:
@noomorph I wonder - shouldn't we be asking ourselves how come `pid` is unavailable, instead of trying to fix?